plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlConfig.java

```diff
@@ -31,6 +31,7 @@
     // implementation, which throw SQL exception when a table disappears during listing.
     // Using `useInformationSchema=false` may provide more diagnostic information (see https://github.com/trinodb/trino/issues/1597)
     private boolean driverUseInformationSchema = true;
+    private boolean enforceTimestampMillisPrecision;
```

Comment:
you may consider adding session property for it so it will be easier to test it. It could be hidden.